HW0/P6/P6B.py

```diff
@@ -11,21 +12,27 @@ def burnTime(t):
     N = 16  # The number of jobs
     P = 4   # The number of processes
 
-    # A thread pool of P processes
-    pool = mp.Pool(P)
+    pool = mp.Pool(P) #create a pool of P processes
 
     # Use a variety of wait times
-    ratio = []
-    wait_time = []
-
+    ratio = [] #empty ratio array to be filled in below
+    #wait_time = np.array([10**-6,10**-5,10**-4,0.5*10**-2,0.5*10**-1,10**-1,1])
+    wait_time = np.array([10**-6,10**-5,10**-4,10**-3,10**-2,10**-1,1])
     for t in wait_time:
         # Compute jobs serially and in parallel
-        # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        begin = time.time()
+        pool.map(burnTime(t),range(N))                        
```

Comment:
thanks, kevin for the note! that may be correct. i think the shape of my curve is right...but i couldn't get the graph to approach 4 (instead of 16 as it is currently doing). I just gave pool.map(burnTime,[1]*16 a try and that seemed to have worked! though i'm not entirely sure of now what the code means..why don't we have to put burnTime(t) anymore, and instead just use burnTime? and what does the brackets around the 1 signify here? :) this was a tricky one for me.